v23.1/create-schedule-for-backup.md

```diff
@@ -95,6 +95,15 @@ We recommend monitoring for your backup schedule to alert for failed backups:
 - See the [Backup and Restore Monitoring](backup-and-restore-monitoring.html) page for a general overview and list of metrics available for backup, scheduled backup, and restore jobs.
 - See [Set up monitoring for the backup schedule](manage-a-backup-schedule.html#set-up-monitoring-for-the-backup-schedule) for metrics and monitoring backup schedules specifically.
 
+### Incremental backup schedules
+
+Scheduled incremental backups only begin after the scheduled full backup is complete. On creation, the first incremental backup waits for the first full backup to finish, then continues at its specified cadence. 
```

Comment:
Suggested reword, feel free to take whatever you think is applicable from it!

The incremental backup schedule is created in a paused state, and is only unpaused on completion of the first, scheduled full backup. This is to ensure that the first incremental backup only begins execution once it has a full backup to build a chain from. Thereafter, the incremental backups will be scheduled at the specified cadence.

Incremental backups always append to the latest, complete full backup. An incremental backup can run concurrently with a full backup, but in such a situation it will continue to append to the previous full backup that has already completed. 

An incremental backup will always wait for another incremental backup started by the same schedule to complete before running. This prevents incremental backups from backing up overlapping timespans in the same backup chain. To enforce this, backup schedules created or altered to set the `on_previous_running` option will have the full backup schedule created with the user specified option, but will **always default** the incremental backup schedule option to `on_previous_running = wait`.